### PR TITLE
ship security_state component

### PR DESCRIPTION
**Pull request by deepak1556**

### CODE REVIEW

File: patches/content_strings.patch [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+index e56c685..4eb9ba2 100644
+--- a/content/app/strings/content_strings.grd
++++ b/content/app/strings/content_strings.grd
+@@ -609,6 +609,38 @@ below:
+         Please enter a number.
+       </message>
+ 
++      <!-- Strings describing Chrome security policy for DevTools security panel -->
++      <message name="IDS_MAJOR_SHA1" desc="Summary phrase for a security problem where the site's certificate expires in 2017 or later and contains a SHA1 signature in the chain." translateable="false">
++        SHA-1 Certificate
```
Comment (zcbenz) [inline patches/content_strings.patch]: We can probably just return raw string in `GetSecurityStyle`.

File: script/create-dist [hunk 2]
```diff
@@ -106,6 +108,7 @@ BINARIES = {
     os.path.join('obj', 'components', 'devtools_http_handler.lib'),
     os.path.join('obj', 'components', 'cdm_renderer.cc.pdb'),
     os.path.join('obj', 'components', 'cdm_renderer.lib'),
+    os.path.join('obj', 'components', 'security_state.lib'),
```
Comment (zcbenz) [inline script/create-dist]: `security_state.cc.pdb` should also be added.

**Comment by deepak1556** [comment]
Have made the changes, thanks!

**Comment by zcbenz** [comment]
👍

**Pull request merged**